src/Plugin/resource/DataProvider/DataProviderResource.php

```diff
@@ -63,7 +63,7 @@ public function __construct(RequestInterface $request, ResourceFieldCollectionIn
    * {@inheritdoc}
    */
   public static function init(RequestInterface $request, $resource_name, array $version) {
-    $plugin_manager = ResourcePluginManager::create();
+    $plugin_manager = ResourcePluginManager::create('cache', $request);
```

Comment:
This is part of the API that exposes `Plug`. This is the bin where the discovered plugins are cached. `'cache'` is the default value.